app/views/admin/users/_member_index.html.erb

```diff
@@ -7,34 +7,40 @@
 
     <!-- Small screen data view start -->
     <div class="block m:hidden fs-s">
-      <h2 class="member-data-heading fs-s py-2 -mx-4 px-6 color-base-60">Members</h2>
-      <ul class="list-none mx-2">
-        <% @users.each do |user| %>
+      <h2 class="member-data-heading fs-s py-2 -mx-3 px-3 color-base-60">Members</h2>
+      <ul class="list-none">
+        <% @users.includes([:organizations]).each do |user| %>
           <li class="py-4">
             <article>
               <header class="flex">
-                <img class="radius-full mr-2" src="<%= user.profile_image_url_for(length: 50) %>" width="40" height="40" alt="" />
-                <div class="flex flex-col">
-                  <h3 class="fs-base"><%= link_to user.name, admin_user_path(user) %></h3>
-                  <span class="color-base-60">@<%= user.username %></span>
+                <a href="<%= admin_user_path(user.id) %>">
+                  <img class="radius-full align-middle" src="<%= user.profile_image_url_for(length: 50) %>" width="40" height="40" alt="<%= user.name %>" />
+                </a>
+                <div class="pl-3 flex-1 fs-s"><%= link_to user.name, admin_user_path(user) %>
+                  <p class="color-base-60">@<%= link_to user.username, admin_user_path(user) %>&nbsp;
+                  </p>
                 </div>
               </header>
               <div class="flex justify-between py-2">
-                <div>standing info</div>
-                <div>role info</div>
+                <div><%= user.current_status %></div>
+                <div>
+                <span class="c-indicator ml-2 crayons-tooltip__activator"><%= user.current_roles %>
+                  <span data-testid="tooltip" class="crayons-tooltip__content"><%= user.current_roles %></span>
+                </span>
```

Comment:
```suggestion
                <span class="c-indicator ml-2 crayons-hover-tooltip" data-tooltip="<%= user.current_roles %>"><%= user.current_roles %>
                </span>
```
I haven't tried this locally yet, but after seeing your comment I think this might solve your tooltip issue! We have this special "hover only" tooltip version... it's not accessible for non-mouse users but this was a known limitation in the designs for this view. Hopefully it's a path of less resistance!

If it doesn't work I'll take a proper look in the morning 🙂

Reply:
Ah, that did the trick! Thank you a MILLION, @aitchiss 🌟